### ISSUE TITLE
tabular data/ noisy instances/ new datasets

### ISSUE THREAD

**Issue by nazaretl** [dbp1994/masters_thesis_codes#1]
Hi,
thanks for sharing your implementation. I have some questions about it:

1.	Does it also work on tabular data?
2.	Is the code tailored to the datasets used in the paper or can one apply it to any data?
3.	Is it possible to identify the noisy instances (return the noisy IDs or the clean set)?

Thanks!

**Comment by dbp1994** [comment]
Hi, 

I think you are referring specifically to the sample-selection algorithm proposed in the thesis. In that case, I am thinking of the following:

1. Can you elaborate on what do you mean by tabular data?
2. I need to polish the code so that it can be used directly. I will do it in some time. Meanwhile, please note that the algorithm only requires softmax values(output of classifier neural network). And the algorithm just computes mean and standard deviation for every mini-batch and selects samples (in that mini-batch) based on that. And this works for any loss function as long as you have softmax values for all the classes per mini-batch. So, it should work for any architecture/dataset/loss function as long as you have access to softmax values. Let me know if there are any difficulties implementing this from your side. I hope this clarifies a bit for you.
3. Yes, just keep track of all the IDs in a given mini-batch during sample selection. Those will be the noisily-labelled samples as deemed by the algorithm.

**Comment by nazaretl** [comment]
thank you for clarifying!
by tabular data I mean non-image data e.g. iris dataset

**Comment by dbp1994** [comment]
I haven't tried it on non-image data but it should work. The proposed algorithm only requires normalised scores/softmax/confidence values. As long as your network can train on those datasets and gives these softmax values as outputs, you can use the proposed sample-selection algorithm.

**Comment by nazaretl** [comment]
I see, thank you!
